### ISSUE TITLE
1.19.1 breaks the client generated code

### ISSUE THREAD

**Issue by YoruNoHikage** [prisma/prisma#3396]
**Describe the bug**
*1.19.1 introduced a breaking change for the updateMany*. I use the javascript generated client but there is a clash between the mutations defined in the generated client and the prisma exposed schema.

**To Reproduce**
1. Update to 1.19.1
2. Re-generate the javascript client
3. try to use `prisma.updateMany...`

**Expected behavior**
It should execute the mutation without error.

**Versions (please complete the following information):**
 - OS: Latest Ubuntu 
 - `prisma` CLI: `prisma/1.19.1 (linux-x64) node-v8.10.0`
 - Prisma Server: `1.19.1`

**Comment by timsuchanek** [comment]
Thanks for reporting @YoruNoHikage ! Can you please post the error here in the issue? That will help us a lot! Thanks!

**Comment by YoruNoHikage** [comment]
I had a type Item and wanted to update a few.

```js
ctx.db.updateManyItems({
  where: { id_in: items.map(item => item.id) },
  data: { status: UPDATED },
})
```

> Variable '$data' of type 'ItemUpdateInput!' used in position expecting type 'ItemUpdateManyMutationInput!'. (line 1, column 35):↵mutation ($where: ItemWhereInput, $data: ItemUpdateInput!)

I just downgrading to 1.19.0 in the meantime.

**Comment by divyenduz** [comment]
@YoruNoHikage : You should be able to use `prisma generate --endpoint` with `1.19.1`. We will fix this issue soon.

**Comment by timsuchanek** [comment]
Thanks for reporting! We released a fix in `1.20.1`.
